BUILD

```diff
@@ -13,27 +18,72 @@ cc_library(
     ],
 )
 
-# TODO(mathetkae): once other runtimes(WAVM,V8) can be linked in this repos,
-#   use -define=wasm=v8|wavm|wasmtime and switch
 cc_library(
-    name = "lib",
-    srcs = glob(
-        [
-            "src/**/*.cc",
-            "src/**/*.h",
-        ],
-        exclude = [
-            "src/**/wavm*",
-            "src/**/v8*",
-        ],
-    ),
-    hdrs = glob(["src/**/*.h"]),
-    copts = COPTS,
+    name = "common_lib",
+    srcs = glob([
+        "src/*.h",
+        "src/*.cc",
+        "src/common/*.h",
+        "src/common/*.cc",
+        "src/third_party/*.h",
+        "src/third_party/*.cc",
+        "src/null/*.cc",
+    ]),
     deps = [
         ":include",
         "@boringssl//:crypto",
         "@com_google_protobuf//:protobuf_lite",
         "@proxy_wasm_cpp_sdk//:api_lib",
+    ],
+)
+
+cc_library(
+    name = "v8_lib",
+    srcs = glob([
+        # TODO(@mathetake): Add V8 lib.
+        # "src/v8/*.h",
+        # "src/v8/*.cc",
+    ]),
+    deps = [
+        ":common_lib",
+        # TODO(@mathetake): Add V8 lib.
```

Comment:
FWIW, it's probably not worth spending time too much time on getting V8 to build, and we can wait until [Bazel is officially supported in V8](https://bugs.chromium.org/p/v8/issues/detail?id=11234).